configure.ac

```diff
@@ -296,7 +296,7 @@
             LUA_LIB_NAME="lua-5.1"
             CFLAGS="${CFLAGS} -DOS_DARWIN"
             CPPFLAGS="${CPPFLAGS} -I/opt/local/include"
-            LDFLAGS="${LDFLAGS} -L/opt/local/lib"
+            LDFLAGS="${LDFLAGS} -L/opt/local/lib -framework Security"
```

Comment:
this is needed by rustls crate...
Should it be commented somewhere ? Where ?

Reply:
what is it? I haven't seen it before

Reply:
Cf https://github.com/cloudflare/quiche/issues/17
Needed to use (link) `_SecRandomCopyBytes` and such

Reply:
ah I see its a mac thing.. missed that initially.

Reply:
Indeed cf `-DOS_DARWIN` :-)